gradle/script/java.gradle

```diff
@@ -27,16 +27,6 @@ jar {
     includeEmptyDirs false
 }
 
-// JDK7 is required for scala 2.9 and 2.10. The scala gradle plugin does not allow changing the java compiler at runtime,
```

Comment:
Is is safe to remove this?

Reply:
Yea, the agent no longer supports Java 7 and won't even start on it.